lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -71,7 +71,7 @@ Value RequestCertificateHandler(const MessageOrigin::Ptr& origin, const Dictiona
 		 * because Icinga versions older than 2.4 sometimes create certificates with an invalid
 		 * serial number. */
 		time_t forceRenewalEnd = 1483228800; /* January 1st, 2017 */
-		time_t renewalStart = now + 30 * 24 * 60 * 60;
+		time_t renewalStart = now + 397 * 24 * 60 * 60;
```

Comment:
The condition this variable is for had the purpose of renewing the certificate if it's valid until <=30 days _from now_ (which was absolutely okay).

What's this change for?